src/pages/components/autocomplete.mdx

```diff
@@ -94,52 +94,96 @@ import SizeCode from '!!raw-loader!../../examples/dropdowns/autocomplete/Size.ts
   </Do>
 </BestPractice>
 
+## Accessibility
+
+If no Label component is provided with the implementation, include an `aria-label` to provide an
+accessible name for the field.
+
```

Comment:
I removed this section entirely so the addition of that section can be agreed upon first. 

My reason for adding this note was that even though we do have an example that shows the ideal way to hide the `Label` accessibly, it is currently entirely possible for teams today to just omit the `Label`. But thinking more about it, a better way might be to throw a dev-only console warning or something if a `Label` is not provided, instead of documenting a non-ideal approach like I was doing there.